socorro/processor/mozilla_transform_rules.py

```diff
@@ -1088,3 +1088,38 @@ def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
             processed_crash
         )
         return True
+
+
+#==============================================================================
+class ThemePrettyNameRule(Rule):
+    """The Firefox theme shows up commonly in crash reports referenced by its
+    internal ID. The ID is not easy to change, and is referenced by id in other
+    software.
+
+    This rule attempts to modify it to have a more identifiable name, like
+    other built-in extensions"""
+
+    #--------------------------------------------------------------------------
+    def __init__(self, config):
+        super(ThemePrettyNameRule, self).__init__(config)
+        self.theme_ext_id = "{972ce4c6-7e08-4474-a285-3208198ce6fd}"
+        self.pretty_ext_id = "default-theme@mozilla.org"
```

Comment:
The fake email address might lead people to think that this is the real name of the theme, and look for it in the Firefox source code. I was imagining something like "{972ce4c6-7e08-4474-a285-3208198ce6fd} (default Firefox theme)" or "Default Firefox theme".